src/message-dispatch.js

```diff
@@ -149,6 +150,7 @@ export default class MessageDispatch extends EventTarget {
             animateLoad: true,
             isObjectMenuTarget: true
           });
+          inflateQuack(APP.world, eid);
```

Comment:
This looks good to me. Another option if you want to implement it as an exercise would be to create a specific prefab for a duck that includes the quack component and here we would be just creating a `Duck` networked entity.

Reply:
Good idea! It feels odd to not address `Duck` at the same time and we might as well expand the learning with this exercise.